scripts/add_electricity.py

```diff
@@ -779,5 +792,11 @@ def add_nice_carrier_names(n, config):
     update_p_nom_max(n)
     add_nice_carrier_names(n, snakemake.config)
 
+    if not ("weight" in n.generators.columns):
+        logger.warning(
+            "Unexpected missing 'weight' column; typical when no generators are detected. Manually added."
+        )
```

Comment:
I'd say rather "it could happen" as "typical" can be misleading. What do you think?

Reply:
Revised